src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs

```diff
@@ -245,7 +245,7 @@ private static void AddPopParams(AuthenticationRequestParameters authenticationR
                     ExpiresIn = (long)(DateTime.SpecifyKind(authResult.ExpiresOn, DateTimeKind.Utc) - DateTimeOffset.UtcNow).TotalSeconds,
                     ClientInfo = authResult.Account.ClientInfo.ToString(),
                     TokenType = authResult.IsPopAuthorization ? Constants.PoPAuthHeaderPrefix : BrokerResponseConst.Bearer,
-                    WamAccountId = authResult.Account.Id,
+                    WamAccountId = authResult.Account.AccountId,
```

Comment:
nit: In the future, would be good to maintain backwards compat to avoid problems if ppl decide to upgarde only 1 package.